cms/static/js/views/course_info_update.js

```diff
@@ -18,31 +19,46 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
                 this.render();
-            // when the client refetches the updates as a whole, re-render them
+                // when the client refetches the updates as a whole, re-render them
                 this.listenTo(this.collection, 'reset', this.render);
                 this.listenTo(this.collection, 'invalid', this.handleValidationError);
             },
 
-            render: function() {
-            // iterate over updates and create views for each using the template
+            resetUpdatesListAndRender: function() {
+                // This method should be used when the course update list needs to be
+                // cleared and then rendered.
                 var updateEle = this.$el.find('#course-update-list');
-            // remove and then add all children
                 $(updateEle).empty();
-                var self = this;
+                this.render();
+            },
+
+            render: function() {
+                // iterate over updates and create views for each using the template
+                var updateEle = this.$el.find('#course-update-list'),
+                    self = this;
                 this.collection.each(function(update, index) {
                     try {
                         CourseInfoHelper.changeContentToPreview(
                         update, 'content', self.options['base_asset_url']);
-                    // push notification is always disabled for existing updates
-                        var newEle = self.template({updateModel: update, push_notification_enabled: false});
-                        $(updateEle).append(newEle);
+                        // push notification is always disabled for existing updates
+                        HtmlUtils.append(
+                            $(updateEle),
```

Comment:
Isn't this already a JQuery element? I don't think it needs to be wrapped with ``$(...)``.